source/wp-content/themes/wporg-developer-2023/src/handbook-pagination/block.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
```

Comment:
Can we not use the filters to output the html? 

https://github.com/WordPress/wporg-developer/blob/dea1a77ba1a665ba25eeffb438d2bb7471ae0d3a/source/wp-content/themes/wporg-developer-2023/functions.php#L190

Reply:
Yes! Great catch, updated in https://github.com/WordPress/wporg-developer/pull/389/commits/e9e7740cf80d021730efadbfd6d3fcda0fa483b0